var/spack/repos/builtin/packages/librsb/package.py

```diff
@@ -11,27 +11,64 @@ class Librsb(AutotoolsPackage):
     library for the Recursive Sparse Blocks format"""
 
     homepage = "http://librsb.sourceforge.net/"
-    url      = "http://download.sourceforge.net/librsb/librsb-1.3.0.0.tar.gz"
+    url      = "http://download.sourceforge.net/librsb/librsb-1.3.0.1.tar.gz"
     list_url = "https://sourceforge.net/projects/librsb/files/"
 
+    version('1.3.0.1',   '3fc024a410f94aca2a7139ae79f4d713b11fa83304293630c363786874c17db4')
     version('1.3.0.0',   '2ac8725d1f988f57df9383ae6b0bb2ed221ec935187d31ebb62ea95ee868a790')
     version('1.2.0.11',  '0686be29bbe277e227c6021de6bd0564e4fc83f996b787886437d28048057bc8')
     version('1.2.0.10',  'ec49f3f78a7c43fc9e10976593d100aa49b1863309ed8fa3ccbb7aad52d2f7b8')
     version('1.2.0.9',   'f421f5d572461601120933e3c1cfee2ca69e6ecc92cbb11baa4e86bdedd3d9fa')
     version('1.2.0.8',   '8bebd19a1866d80ade13eabfdd0f07ae7e8a485c0b975b5d15f531ac204d80cb')
 
     depends_on('zlib')
+    depends_on('googletest', type='build', when='+googletest')
     conflicts('%apple-clang')
-    conflicts('%clang')
+    # conflicts('%clang')
+    conflicts('%gcc@11.0.0:11.2.99', msg='gcc-11.0:gcc-11.3 can break librsb on x86_64')
+
+    variant('asan', default=False, description="Use ASAN.")
+    variant('debug', default=False, description="Enable debug features.")
+    variant('googletest', default=False, description="Use Google Test as prerequisite.")
+    variant('native', default=True, description="Use native flags.")
+    variant('nospblas', default=False, description="Disable Building The Sparse BLAS API.")
+    variant('serial', default=False, description="Disable OpenMP support.")
+    variant('verbose', default=False, description="Extra Library Verbosity. Good for learning.")
+
+    conflicts('+asan', when='+native', msg='native must be disabled when asan is enabled')
```

Comment:
Please move this to be co-located with the other `conflicts`.  That will help with readability and maintainability in the long run.

Reply:
If that's the Spack project style, I'm eager to follow that. Here the changes.